sources/packages/web/src/components/common/notes/Notes.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <v-row class="m-2">
+    <v-col class="category-header-medium color-blue">{{ title }}</v-col>
+    <v-col
+      ><v-btn @click="addNewNote()" class="float-right primary-btn-background">
+        <v-icon left dark size="25"> mdi-open-in-new </v-icon>Create new
+        note</v-btn
+      ></v-col
+    >
+  </v-row>
+  <v-row class="m-2" v-if="!notes || notes.length === 0"
+    ><v-col
+      >No notes found. Please click on create new note to add one.</v-col
+    ></v-row
+  >
+  <Timeline :value="notes">
+    <template #content="slotProps">
+      <v-row>
+        <v-col cols="2" class="primary-color marker-text">{{
+          dateOnlyLongString(slotProps.item.createdAt)
+        }}</v-col>
+        <v-col>
+          <div class="content-header">{{ slotProps.item.noteType }}</div>
+          <div v-if="showMoreNotes(slotProps.item)" class="header mt-2">
+            {{ slotProps.item.description.substring(0, 150) }}
+            <a
+              @click="toggleNotes(slotProps.item)"
+              target="_blank"
```

Comment:
why `target="_blank"` for this show more.
![image](https://user-images.githubusercontent.com/77353155/145634751-433485a7-9bdc-4a48-97c7-8915258aa3b5.png). is this the requirement, I am not understanding `target="_blank"` in line 38 too. both of them are not making sense to me.

Reply:
That piece of code is removed.